Can not verify signatures after following instructions in "Getting Started Tutorial"

# Expected Behavior

I'm currently investigating chains and installed it on my local cluster and tried the instruction from the ["Getting Started Tutorial"](https://tekton.dev/docs/chains/getting-started-tutorial/) and I'm not able to successfully verify the resulting payload.

Specifically running:
```
cosign verify-blob --key k8s://tekton-chains/signing-secrets --signature ./signature ./payload
``` 
returns:
```
Error: verifying blob [./payload]: failed to verify signature                                                                                         
main.go:46: error during command execution: verifying blob [./payload]: failed to verify signature
````
not 
```
Verified OK

```

This may be a PEBKAK error, but the documentation on how to debug this is a little thin.

The only thing I can think I've done differently then just blindly follow the installation guide and then the tutorial is I loaded the following `ConfigMap`, though I believe I'm basically specifying some things to default values.

```
apiVersion: v1                                                                                                                                        
kind: ConfigMap                                                                                                                                       
metadata:                                                                                                                                             
  labels:                                                                                                                                             
    app.kubernetes.io/instance: default                                                                                                               
    app.kubernetes.io/part-of: tekton-chains                                                                                                          
  name: chains-config                                                                                                                                 
  namespace: tekton-chains                                                                                                                            
data:                                                                                                                                                 
   artifacts.taskrun.format: in-toto                                                                                                                  
   artifacts.taskrun.storage: tekton                                                                                                                  
   artifacts.pipelinerun.format: in-toto                                                                                                              
   artifacts.pipelinerun.storage: tekton                                                                                                              
   artifacts.oci.storage: tekton                                                                                                                      
   builder.id: http://tekton.internal.jpace121.net
```

# Actual Behavior
```
cosign verify-blob --key k8s://tekton-chains/signing-secrets --signature ./signature ./payload
```
returns `Error: verifying blob [./payload]: failed to verify signature                                                                                         
main.go:46: error during command execution: verifying blob [./payload]: failed to verify signature`.

# Steps to Reproduce the Problem

Blindly follow instructions https://tekton.dev/docs/chains/getting-started-tutorial/ with the exception of using the config map specified above.
Verify the payload signature.

# Additional Info

- Kubernetes version:
```
WARNING: This version information is deprecated and will be replaced with the output from kubectl version --short.  Use --output=yaml|json to get the full version.
Client Version: version.Info{Major:"1", Minor:"26", GitVersion:"v1.26.0", GitCommit:"b46a3f887ca979b1a5d14fd39cb1af43e7e5d12d", GitTreeState:"clean", BuildDate:"2022-12-08T19:58:30Z", GoVersion:"go1.19.4", Compiler:"gc", Platform:"linux/amd64"}
Kustomize Version: v4.5.7
Server Version: version.Info{Major:"1", Minor:"24", GitVersion:"v1.24.4+k3s1", GitCommit:"c3f830e9b9ed8a4d9d0e2aa663b4591b923a296e", GitTreeState:"clean", BuildDate:"2022-08-25T03:45:26Z", GoVersion:"go1.18.1", Compiler:"gc", Platform:"linux/amd64"}
WARNING: version difference between client (1.26) and server (1.24) exceeds the supported minor version skew of +/-1
  ```

- Tekton Pipeline version:

```
$ tkn version
Client version: 0.25.0
Chains version: v0.14.0
Pipeline version: v0.39.0
Triggers version: v0.20.2
Dashboard version: v0.30.0

```
I'm not  sure if this is relevant, but the version of cosign on my laptop is:
```
$ cosign version
GitVersion:    v1.6.0
GitCommit:     4b2c3c0c8ee97f31b9dac3859b40e0a48b8648ee
GitTreeState:  clean
BuildDate:     '2022-03-03T17:59:06Z'
GoVersion:     go1.17.7
Compiler:      gc
Platform:      linux/amd64
```

--- a/docs/tutorials/getting-started-tutorial.md
+++ b/docs/tutorials/getting-started-tutorial.md
@@ -82,14 +82,13 @@ base64-encoded annotations):
 
 ```shell
 export TASKRUN_UID=$(tkn tr describe --last -o  jsonpath='{.metadata.uid}')
-tkn tr describe --last -o jsonpath="{.metadata.annotations.chains\.tekton\.dev/signature-taskrun-$TASKRUN_UID}" > signature
-tkn tr describe --last -o jsonpath="{.metadata.annotations.chains\.tekton\.dev/payload-taskrun-$TASKRUN_UID}" | base64 -d > payload
+tkn tr describe --last -o jsonpath="{.metadata.annotations.chains\.tekton\.dev/signature-taskrun-$TASKRUN_UID}" | base64 -d > sig
 ```
 
 Finally, we can check the signature with [cosign]:
 
 ```shell
-$ cosign verify-blob --key k8s://tekton-chains/signing-secrets --signature ./signature ./payload
+$ cosign verify-blob --key k8s://tekton-chains/signing-secrets --signature sig sig
 Verified OK
 ```
 

